src/form/wr.jl

```diff
@@ -36,23 +36,23 @@ p[f_idx] ==        g/tm*w_fr[i] + (-g*tr+b*ti)/tm*(wr[i]) + (-b*tr-g*ti)/tm*(wi[
 q[f_idx] == -(b+c/2)/tm*w_fr[i] - (-b*tr-g*ti)/tm*(wr[i]) + (-g*tr+b*ti)/tm*(wi[i])
 ```
 """
-function constraint_ohms_tp_yt_from_on_off(pm::GenericPowerModel{T}, n::Int, h::Int, i, f_bus, t_bus, f_idx, t_idx, g, b, g_fr, b_fr, tr, ti, tm, vad_min, vad_max) where T <: PMs.AbstractWRForm
-    p_fr = var(pm, n, h, :p, f_idx)
-    q_fr = var(pm, n, h, :q, f_idx)
+function constraint_ohms_tp_yt_from_on_off(pm::GenericPowerModel{T}, n::Int, c::Int, i, f_bus, t_bus, f_idx, t_idx, g, b, g_fr, b_fr, tr, ti, tm, vad_min, vad_max) where T <: PMs.AbstractWRForm
+    p_fr = var(pm, n, c, :p, f_idx)
+    q_fr = var(pm, n, c, :q, f_idx)
     w    = var(pm, n, :w)
     wr   = var(pm, n, :wr)
     wi   = var(pm, n, :wi)
 
-    @constraint(pm.model, p_fr ==  ( g_fr[h]+g[h,h]) * w[(f_bus, h)] +
-                                sum( g[h,i] * wr[(f_bus, f_bus, h, i)] +
-                                     b[h,i] * wi[(f_bus, f_bus, h, i)] for i in PMs.phase_ids(pm) if i != h) +
-                                sum(-g[h,i] * wr[(f_bus, t_bus, h, i)] +
-                                    -b[h,i] * wi[(f_bus, t_bus, h, i)] for i in PMs.phase_ids(pm)) )
-    @constraint(pm.model, q_fr == -( b_fr[h]+b[h,h]) * w[(f_bus, h)] -
-                                sum( b[h,i] * wr[(f_bus, f_bus, h, i)] -
-                                     g[h,i] * wi[(f_bus, f_bus, h, i)] for i in PMs.phase_ids(pm) if i != h) -
-                                sum(-b[h,i] * wr[(f_bus, t_bus, h, i)] +
-                                     g[h,i] * wi[(f_bus, t_bus, h, i)] for i in PMs.phase_ids(pm)) )
+    @constraint(pm.model, p_fr ==  ( g_fr[c]+g[c,c]) * w[(f_bus, c)] +
+                                sum( g[c,i] * wr[(f_bus, f_bus, c, i)] +
+                                     b[c,i] * wi[(f_bus, f_bus, c, i)] for i in PMs.conductor_ids(pm) if i != c) +
+                                sum(-g[c,i] * wr[(f_bus, t_bus, c, i)] +
+                                    -b[c,i] * wi[(f_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm)) )
+    @constraint(pm.model, q_fr == -( b_fr[c]+b[c,c]) * w[(f_bus, c)] -
+                                sum( b[c,i] * wr[(f_bus, f_bus, c, i)] -
+                                     g[c,i] * wi[(f_bus, f_bus, c, i)] for i in PMs.conductor_ids(pm) if i != c) -
+                                sum(-b[c,i] * wr[(f_bus, t_bus, c, i)] +
+                                     g[c,i] * wi[(f_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm)) )
```

Comment:
may want to change these `i` to `d` in the constraint sums as well so they don't collide with the `i` in the function arguments.